news/1651.misc

```diff
@@ -0,0 +1 @@
+Added getters for several BedrockLandslider input parameters.
```

Comment:
Just for fun, instead of `BedrockLandslider`, write `` :class:`~.BedrockLandslider` `` so that it will link to its documentation page.

Reply:
Great tip! Done.